lib/constructs/interface.js

```diff
@@ -317,11 +446,11 @@ class Interface {
         return `${value} !== ${unsupportedValue}`;
       }
       return `${O}[impl][utils.supportsPropertyName](${P})`;
-    }
+    };
 
     // "named property visibility algorithm"
     // If `supports` is true then skip the supportsPropertyName check.
-    function namedPropertyVisible(P, O, supports = false) {
+    const namedPropertyVisible = (P, O, supports = false) => { // eslint-disable-line func-style
```

Comment:
Nah, I prefer function declarations for functions, and arrow functions for cases where the function is being passed somewhere.